tests/plugin/common.py

```diff
@@ -0,0 +1,89 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Common methods for plugin tests"""
+
+from contextlib import contextmanager
+from unittest import mock
+
+import allure
+import allure_commons
+from _pytest.pytester import Pytester, RunResult
+from allure_commons.logger import AllureMemoryLogger
+
+
+@contextmanager
+def fake_logger(mock_path, logger):
+    """Fake Allure logger"""
+
+    blocked_plugins = []
+    for name, plugin in allure_commons.plugin_manager.list_name_plugin():
+        allure_commons.plugin_manager.unregister(plugin=plugin, name=name)
+        blocked_plugins.append(plugin)
+
+    with mock.patch(mock_path) as ReporterMock:  # pylint: disable=invalid-name
```

Comment:
Naming convention violation. Use snake_case for variable naming

Reply:
Target of mock is a class object therefore I think it's better to keep CamelCase.